samples/confluent-schema-registry-integration/confluent-schema-registry-integration-producer1/pom.xml

```diff
@@ -15,24 +15,20 @@
 	</parent>
 
     <dependencies>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-actuator</artifactId>
-        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-        <dependency>
```

Comment:
I just moved all of the avros below, together in the 3 app poms.

Reply:
LGTM.